src/extension.ts

```diff
@@ -3,19 +3,46 @@ import * as vscode from 'vscode';
 let on = true;
 let count = 1;
 
+function getSassyMessage() {
+  // we could show messages less frequently
+  if (count < 10) {
+    return `You have used the mouse ${count} times`;
+  } else if (count < 20) {
+    return `You might be a heavy mouse user, already used the mouse ${count} times.`;
+  } else if (count === 20) {
+    return `This isn't working. It's me, it's not you. Maybe turn it off?`;
+  } else if (count < 50) {
+    return `Really, mouse? Again? (failure count: ${count})`;
```

Comment:
Not a big fan of this `(failure count: ${count})` thing anymore since the tone/language of the other messages have changed.